specification/iotcentral/resource-manager/readme.ruby.md

```diff
@@ -12,10 +12,21 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
+  - tag: package-2020-03-01
```

Comment:
Is this API change required since it's just an SDK breaking change and not an RP

Reply:
ok